ui/lib/sync/addon/components/secrets/page/destinations/destination/sync.hbs

```diff
@@ -13,15 +14,21 @@
   }}
 />
 
-<form {{on "submit" (perform this.setAssociation)}} class={{unless (or this.error this.syncedSecret) "has-top-margin-m"}}>
+<form {{on "submit" (perform this.setAssociation)}}>
   <MessageError @errorMessage={{this.error}} />
 
   {{#if this.syncedSecret}}
-    <Hds::Alert @type="inline" @color="success" as |A|>
-      <A.Title>Successfully synced a secret</A.Title>
+    <Hds::Alert @type="inline" @color="success" @icon="sync" as |A|>
+      <A.Title>Sync initiated</A.Title>
       <A.Description data-test-sync-success-message>
-        Sync operation successfully initiated for "{{this.syncedSecret}}". You can continue on this page to sync more
-        secrets.
+        Sync operation successfully initiated for
+        <Hds::Link::Inline
```

Comment:
good catch

Reply:
This wasn't in the designs but I thought it was a nice addition!